app/mint/page.tsx

```
@@ -81,12 +81,19 @@ export default function MintPage() {
   const [activeTab, setActiveTab] = useState<'mint' | 'burn' | 'rates'>('mint');
   const [step, setStep] = useState<'input' | 'confirm' | 'success'>('input');
   const [burnAmount, setBurnAmount] = useState('');
+  const [burnError, setBurnError] = useState('');
+  const [rates, setRates] = useState<RatesResponse | null>(null);
```

ESLint

[error] The value assigned to 'rates' is not used in subsequent statements.

(no-useless-assignment)